docs/tutorial/10-resources-compose.mdx

```diff
@@ -133,7 +133,7 @@ These definitions show how a Kitty resource could own hats.
 The hats are stored in a variable in the Kitty resource.
 
     // place where the Kitty hats are stored
-    pub let items: <-{String: KittyHat}
+    `pub let items: <-{String: KittyHat}`
 
```

Comment:
```suggestion
    pub let items: <-{String: KittyHat}
```
```

Reply:
This needs to be in a code fence, since MDX2 will interpret the { } as an JS expression template, and fail to render the page.

MDX 2 expects anything inside { } to be valid JS and There is no way to turn this off afaik.

Reply:
Yep, see above. My suggestion was to wrap it in a block code fence (```), instead of an inline one (`)